general-knowledge/about-mycrypto/membership-information.md

```diff
@@ -30,10 +30,10 @@ This is just the beginning, but right now being a MyCrypto Member gives you:
 We have a lot of plans for the MyCrypto Membership and a long-term roadmap that will see many new rewards and features coming soon.
 
 ### How much does the MyCrypto Membership cost?
-$4.00 per month, or you can purchase multiple month packages at a discounted rate. For the hardcore, there’s also a lifetime membership.
+$4.00 per month, or you can purchase multiple month packages at a discounted rate. For the hardcore, there’s also a lifetime membership. Please keep in mind that at this time, it's only possible to purchase monthly memberships with DAI.
```

Comment:
Please keep in mind that at this time, it's only possible to purchase monthly memberships with DAI. We may add more options in the future.

Reply:
Ty!